common/app/views/fragments/email/stylesheets/front.scala.html

```diff
@@ -45,14 +45,17 @@
         text-decoration: none;
     }
     .facia-card__text {
-        padding: 0 10px 26px 10px !important;
+        padding: 0 10px 0px 10px !important;
```

Comment:
Sadly the email-specific CSS framework we use has `!important` all over the place so we inevitably have to use it too to override. I think it's because of style inlining - Gmail strips all `<style>` blocks out so all styles which can be inlined (not `:hover`, for example) are stuffed into `style` attributes on the relevant elements. But for styles which get left behind in the `<head>` and are left intact in other clients, `!important` is needed to make them take effect over the inlined styles